asmrun/startup.c

```diff
@@ -74,8 +74,10 @@ static void init_segments(void)
 }
 
 /* These are termination hooks used by the systhreads library */
```

Comment:
Probably not the most friendly way to remove this logic. Should properly delete this logic if it's really obsolete.